src/transform.ts

```diff
@@ -106,3 +110,27 @@ function renameFields(records: DataRecord, renameFieldsObj: Record<string, strin
     });
     return renamedRecords as DataRecord;
 }
+
+function mapDataToInternationalFormat(records: DataRecord, resource: TransformableResource) {
```

Comment:
Maybe lets rename this function to something like `mapGermanToInternationalNumberFormat`? Since the implementation is specific to German (there are other formats like French out there)

Reply:
Good point